code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -1,4 +1,5 @@
 #define EMPOWERED_THRALL_LIMIT 5
+var/in_jaunt = 0
```

Comment:
There absolutetely no need for this. I recommend using the part I did in my old nerf PR where I simply used ''incorporeal_movement'' to check if the Sling was in jaunt. This would be simplier and there would be no need for this global var.